### DISCUSSION TITLE
No setup?

### DISCUSSION THREAD

**Discussion by eggbean** [WhitewaterFoundry/Fedora-Remix-for-WSL#204] [Q&A] [answered]
I've been using WLinux (now Pengwin) for a few years and I recently bought Fedoraremix thinking it would be a good way to get familiar with btrfs.

One of the good things about using Pengwin is the setup script which sets up a few things, like the graphical desktop.

When starting Fedoraremix I cannot see any welcome message or indication that there is a similar setup script. It also starts as the root user.

Is that it? I will have to do everything manually?

**Comment by crramirez** [comment, marked as answer]
Hello @eggbean,

Fedora Remix for WSL we don't include pengwin-setup with fedora remix. And yes, you'll have to do almost everything manually. But Fedora Remix includes automatic GPU acceleration, integration with XServers if you don't have WSLg also built in SystemD support if you don't want or can't use the WSL store version.

It should not start with root user, probably you had a problem with the installation. I recommend deregister the distro with `wsl.exe --unregister fedoraremix` and start again. It will ask for a username and password as Pengwin does.

Here you can see a step by step instructions on how to install Gnome Desktop.

I believe that using btrfs as the root filesystem in WSL2 is very difficult but you can create another volume and mount it.

https://learn.microsoft.com/en-us/windows/wsl/wsl2-mount-disk

Hope it helps

**Reply by eggbean** [reply]
Thanks for your help. Where are those instructions on installing the GNOME desktop though?

Ah, bit of a pity about the the root partition being ext4 :/

**Reply by crramirez** [reply]
Sorry I forgot to include them https://www.whitewaterfoundry.com/blog/2022/4/16/announcing-built-in-systemd-support-in-fedora-remix-for-wsl
